manifests/charts/istiod-remote/values.yaml

```diff
@@ -33,13 +33,13 @@ sidecarInjectorWebhook:
   objectSelector:
     enabled: false
     autoInject: true
-  
-  # remote webhook host and port number to point the mutating webhook configuration to.
-  remoteWebhookHost: istiod.istio-system.svc
-  remoteWebhookPort: 443
 
   # caBundle to be patched at runtime or user can specify it manually
   caBundle: ""
 
+  # Sidecar injector mutating webhook configuration url
+  # For example: https://$remotePilotAddress:15053/inject
+  injectionURL: ""  
```

Comment:
My previous comment was that we want to stop adding values.yaml based config - we have 300 or so already that need to be migrated to mesh config or proper APIs. For new ones - we should go directly to mesh config, at least if they are not experiments ( and this doesn't look like one ). 

I think mesh config already defines a number of 'services' ( tracing, etc) - maybe we should use the same pattern there. Or add them consistently.

Reply:
Ok I see, I will look into adding them to meshconfig in a followup PR.  I've removed the proto change.